lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -0,0 +1,31 @@
+module GovukIndex
+  class IndexableContentPresenter
+    def initialize(details)
+      @details = details
+      @sanitiser = IndexableContentSanitiser.new
+    end
+
+    def indexable_content
+      return nil if details.nil?
+      sanitiser.clean(elements)
+    end
+
+  private
+
+    attr_reader :details, :sanitiser
+
+    def elements
+      if details['parts']
+        parts(details)
```

Comment:
The top level page for something with parts will start with one of the parts. This seems to be the overview from all the examples I have found. Each part is always in the `details` `parts` payload